packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -796,11 +804,15 @@ class Scaffold extends StatefulWidget {
     this.resizeToAvoidBottomPadding,
     this.resizeToAvoidBottomInset,
     this.primary = true,
+    this.extendedBody = false,
     this.drawerDragStartBehavior = DragStartBehavior.down,
   }) : assert(primary != null),
        assert(drawerDragStartBehavior != null),
        super(key: key);
 
+  /// When "true" - the body fills the entire height of Scaffold
```

Comment:
If true and a [bottomNavigationBar] is specified then the body extends to the bottom of the Scaffold, instead of only extending to the top of the [bottomNavigationBar].

Reply:
done